ratis-common/src/main/java/org/apache/ratis/protocol/TransferLeadershipRequest.java

```diff
@@ -19,14 +19,21 @@
 
 public class TransferLeadershipRequest extends RaftClientRequest {
   private final RaftPeerId newLeader;
+  private final long timeoutInMs;
```

Comment:
Let's add it to RaftClientRequest.  It would be useful for the other requests.

Also, let's just name it "timeoutMs".